src/diffusers/loaders/peft.py

```diff
@@ -31,6 +31,7 @@
 _SET_ADAPTER_SCALE_FN_MAPPING = {
     "UNet2DConditionModel": _maybe_expand_lora_scales,
     "SD3Transformer2DModel": lambda model_cls, weights: weights,
+    "UNetMotionModel": lambda model_cls, weights: weights,
```

Comment:
I think `_maybe_expand_lora_scales` was applied in the `UNetMotionModel` since that was the default in `UNet2DConditionalLoaderMixin` 

https://github.com/huggingface/diffusers/blob/c586aadef6bb66d355fa40a2b95a0bea8a6fe79c/src/diffusers/loaders/unet.py#L620

Reply:
Thanks! Fixed in 2cbcc24df.